masterlist.yaml

```
@@ -1672,6 +1672,17 @@ plugins:
     group: *fixesGroup
     msg: [ *alreadyInUFO4P ]
 
+  - name: 'DisabletheCaravan.esl'
+    url:
+      - link: 'https://www.nexusmods.com/fallout4/mods/43961'
+        name: 'Covenant - Disable the dead caravan from Stockton'
+    group: *fixesGroup
+    dirty:
```

To go with the name change above as this cleaning data only applies to the ESL:

```suggestion
  - name: 'DisabletheCaravan.esl'
    dirty:
```